internal/gps/result_test.go

```diff
@@ -35,12 +34,6 @@ func init() {
 			}, nil),
 		},
 	}
-
-	// just in case something needs punishing, kubernetes is happy to oblige
-	kub = atom{
-		id: pi("github.com/kubernetes/kubernetes"),
-		v:  NewVersion("1.0.0").Is(Revision("528f879e7d3790ea4287687ef0ab3f2a01cc2718")),
```

Comment:
Is dead code the best way to document that choice? What about commenting out the block?

Is there any other rationale for this particular revision, other than that its 'known to work'?
```go
// Just in case something needs punishing, kubernetes offers a complex,
// real-world set of dependencies, and this revision is known to work.
/*
	_ = atom{
		id: pi("github.com/kubernetes/kubernetes"),
		v:  NewVersion("1.0.0").Is(Revision("528f879e7d3790ea4287687ef0ab3f2a01cc2718")),
	}
*/
```